gunicorn/config.py

```diff
@@ -953,6 +957,18 @@ class Sendfile(Setting):
            added support for the ``SENDFILE`` environment variable
         """
 
+class ReusePort(Setting):
+    name = "reuse_port"
+    section = "Server Mechanics"
+    cli = ["--reuse-port"]
+    validator = validate_bool
+    action = "store_true"
+    default = False
+
+    desc = """\
+        Set the ``SO_REUSEPORT`` flag on the listening socket.        
```

Comment:
Please add `.. versionadded:: 19.8`.

Reply:
I always forget this and you always catch it. Thanks! :-D